legacy/PimRecordListHelper.js

```diff
@@ -53,11 +55,19 @@ async function PimRecordListHelper(
   );
 
   // filter the records if rows were selected or filters applied in product list page
-  let filteredRecords = exportRecords.filter(
-    record =>
-      recordIds.includes(record.get('Id')) ||
+  let filteredRecords;
```

Comment:
The logic is a bit different, if exportType is 'lowestVariants' then there shouldn't be any ids from recordIds in filteredRecords. So it would have to be 

```
filteredRecords = exportRecords.filter(
      record =>
        (exportType === 'lowestVariants' && variantValueIds?.includes(record.get('Id'))) || 
        (exportType !== 'lowestVariants' && (recordIds.includes(record.get('Id'))) || variantValueIds?.includes(record.get('Id'))))
);
```
which I find quite unreadable

Reply:
oops, on the logic. but while i agree it is harder to read, that is why the variables are there to abstract the logic. This way the code is shorter and it reads easier since the variable names tell you what the logic is. trying to do